### PR TITLE
Reverse word in a String

### LINKED ISSUE #1921: C/CPP Reversing a word in a string
### Description

Documentation on how reverse a word  in a string

### Domain

C/CPP

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by yellowberard**
<hr>

## Description 📜

I have added a solution for reversing word in a string 
Fixes #1921

<hr>

## Type of change 📝

- [x] Doc#umentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] C/#CPP
<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by yellowberard** [comment]
@2024-SANDHYA  please review my PR

File: C_CPP/Strings and their functions/reverse_string.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<h1 text align="center"> Reversing words in a String</h1>
```
Comment (apeksha235) [inline C_CPP/Strings and their functions/reverse_string.md:1]: please use [markdown](https://www.markdownguide.org/basic-syntax/)  formatting and not html

File: C_CPP/Strings and their functions/reverse_string.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+<p>This a answer for leetcode  <a href"https://leetcode.com/problems/reverse-words-in-a-string/">question</a> https://leetcode.com/problems/reverse-words-in-a-string/ <br>
```
Comment (apeksha235) [inline C_CPP/Strings and their functions/reverse_string.md:2]: write down the problem statement, don't provide a link  for it

File: C_CPP/Strings and their functions/reverse_string.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+Example<br>  Input: s = "a good   example" <br>
```
Comment (apeksha235) [inline C_CPP/Strings and their functions/reverse_string.md:3]: add the input output inside ```

File: C_CPP/Strings and their functions/reverse_string.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+ Output: "example good a"
+<br><h2>Solution</h2>
+
+
+```
```
Comment (apeksha235) [inline C_CPP/Strings and their functions/reverse_string.md:8]: use ```c++ syntax highlighting

**Review by apeksha235 — CHANGES_REQUESTED** [review]
@yellowberard please make these changes

File: C_CPP/Strings and their functions/reverse_string.md (hunk 2 above)
Comment (kashishahuja2002) [inline C_CPP/Strings and their functions/reverse_string.md:2]: Embed the link in the name of the problem question and also write the whole problem here.

File: C_CPP/Strings and their functions/reverse_string.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    reverse(word.begin(),word.end());  //reversing the word so that it comes to original form
+    rev_word += word; //storing word
+    rev_word += " ";     //adding space
+}
+
+rev_word.erase(rev_word.length()-1,1); //removing the additional last space added 
+return rev_word; 
+}
+};
+```
```
Comment (kashishahuja2002) [inline C_CPP/Strings and their functions/reverse_string.md:28]: Add proper indentation and comments to the code.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
Also, add your approach to solving the problem.

**Comment by yellowberard** [comment]
Can I make a new PR because of some issue with github I had to refork the repository ??

**Comment by yellowberard** [comment]
@kashishahuja2002

**Comment by kashishahuja2002** [comment]
> Can I make a new PR because of some issue with github I had to refork the repository ??

Refork, don't know why you needed to do that. Try solving issues in the same PR. If not then you can close this and make a new one.

**Pull request closed without merging**